sw/device/lib/crypto/drivers/aes.c

```diff
@@ -0,0 +1,158 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/crypto/drivers/aes.h"
+
+#include "sw/device/lib/base/bitfield.h"
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/base/mmio.h"
```

Comment:
I thought the cryptolib drivers were going to use `abs_mmio` (the existing hmac driver does).

Reply:
Yes, we've agreed on `abs_mmio` for the cryptolib drivers -- I moved it out of `silicon_creator` to `lib/base`, so it should be pretty straightforward now.

Reply:
(I updated to use abs_mmio... I am not a fan, personally, since for a constant mmio_region it compiles to the same thing... but meh.)